lib/reform/form/validation/accept_validator.rb

```diff
@@ -0,0 +1,22 @@
+# Rails acceptance validation that does not write to model.
+class Reform::Form::AcceptValidator < ActiveModel::EachValidator
+  def initialize(options)
+    super({ accepted: ["1", true] }.merge!(options))
+  end
+
+  def validate_each(form, attribute, value)
+    unless accepted_option?(value)
+      form.errors.add(attribute, :accepted, options)
```

Comment:
I wasn't sure what to use here as a message. In `UniqueValidator` this message is hardcoded, but we still inherit from `ActiveModel::EachValidator` so as for me it makes more sense to use I18n key.

Let me know if you want "key version" or "hardcoded message". I can also update `UniqueValidator` if you decide in favor of "key version".